src/generateAppendClassName.js

```diff
@@ -1,4 +1,4 @@
-import Map from 'es6-map';
+import Map from './simple-map';
```

Comment:
The reason I have suggested "weak map" is because keys are objects. @benjamingr Can you suggest a better name?